UploadCSV/Program.cs

```diff
@@ -14,38 +14,21 @@ static class Program
         #region IMPORT CSV FILE TO THE DATABASE
         static void Main(string[] args)
         {
-            var connectionString = ConfigurationManager.ConnectionStrings["csvFileContext"].ConnectionString;
-            using (SqlConnection connection = new SqlConnection(connectionString))
+            try
             {
-                try
+                using (Spreadsheet document = new Spreadsheet())
                 {
-                    connection.Open();
-                    #region DATABASE QUERY
-                    //// Drop test database if exists
-                    //ExecuteQueryWithoutResult(connection, "IF DB_ID('CSV_FILE_TEST') IS NOT NULL DROP DATABASE CSV_FILE");
-                    //// Create empty database
-                    //ExecuteQueryWithoutResult(connection, "CREATE DATABASE CSV_FILE_TEST");
+                    document.LoadFromFile(ConfigurationManager.AppSettings["path"], ","); //path of the file
+                    Worksheet worksheet = document.Workbook.Worksheets[0];
 
-                    // Switch to created database
-                    //ExecuteQueryWithoutResult(connection, "USE CSVtoDB"); // change "CSVtoDB" to database name
-
-                    //// Create a table for CSV data
-                    //ExecuteQueryWithoutResult(connection,
-                    //"CREATE TABLE [dbo].[cr_expences](iCtr INT,cCRCode CHAR(50),cExpenses CHAR(50),iCost DECIMAL(9, 2))");
-                    #endregion
-
-                    using (Spreadsheet document = new Spreadsheet())
+                    try
                     {
-                        //Console.WriteLine("Please wait...");
-
-                        document.LoadFromFile(ConfigurationManager.AppSettings["path"], ","); //path of the file
-
-                        Worksheet worksheet = document.Workbook.Worksheets[0];
-
-                        try
+                        for (int row = 0; row <= worksheet.UsedRangeRowMax; row++)
                         {
-                            for (int row = 0; row <= worksheet.UsedRangeRowMax; row++)
+                            var connectionString = ConfigurationManager.ConnectionStrings["csvFileContext"].ConnectionString;
```

Comment:
Paki labas sa for loop, di natin need ideclare yung connection string ng paulit ulit